src/sentry/api/event_search.py

```diff
@@ -171,6 +172,7 @@ def translate(pat):
 alt_date_format      = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?(\+\d{2}:\d{2})?)?(?=\s|\)|$)"
 rel_date_format      = ~r"[\+\-][0-9]+[wdhm](?=\s|\)|$)"
 duration_format      = ~r"([0-9\.]+)(ms|s|min|m|hr|h|day|d|wk|w)(?=\s|\)|$)"
+percentage_format    = ~r"([0-9\.]+)%"
```

Comment:
This will allow `0.0.0.0`. I don't think that's what you want. The pattern of `(?:[0-9]+)?\.(?:[0-9]+)` would prevent that.

Reply:
🙃 oh nice `duration_format` will allow `0.0.0.0.0ms` too I'll update that as well

Reply:
_annnnd_ `numeric_value` (which is what I based `duration_format` on) 🙃 🙃 🙃

Reply:
Okay, going to keep this as is, the error message of `*:5.5.5.5.5%` becomes `5.5.5.5 is not a valid percentage value`. Improving this regex results in the query being parsed as a message instead.